cms/static/js/views/pages/library_container.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * PagedXBlockContainerPage is a variant of XBlockContainerPage that supports Pagination.
+ */
+define(["jquery", "js/views/pages/paged_container", "js/views/library_container"],
+    function ($, PagedContainerPage, LibraryContainerView) {
+        'use strict';
+        var LibraryContainerPage = PagedContainerPage.extend({
+
+            viewClass: LibraryContainerView,
+
+            events: {
+                'click .new-component-button': 'scrollToNewComponentButtons',
+            },
+
+            scrollToNewComponentButtons: function(event) {
```

Comment:
I'm wondering why this is specific to the library container, instead of just being part of PagedContainerPage. Can you explain?